Makefile

```diff
@@ -109,8 +109,6 @@ lint-rust:
 	cargo clippy --tests -- \
 		-D warnings \
 		-W clippy::pedantic \
-		-W clippy::dbg_macro \
-		-W clippy::print_stdout \
 		-A clippy::cast-possible-truncation \
 		-A clippy::cast-possible-wrap \
```

Comment:
I wasn't able to move the rest of these lints to the `Cargo.toml`, see https://github.com/rust-lang/rust-clippy/issues/11830